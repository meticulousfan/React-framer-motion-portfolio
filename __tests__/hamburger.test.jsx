/**
 * @jest-environment jsdom
 */

import React from "react";
import { render } from "@testing-library/react";

import Hamburger from "../src/components/UI/Hamburger.component";

describe("Hamburger", () => {
  it("Hamburger laster inn og kan vises", () => {
    const { hamburger } = render(<Hamburger />);
    expect(hamburger).toMatchSnapshot();
  });
});
