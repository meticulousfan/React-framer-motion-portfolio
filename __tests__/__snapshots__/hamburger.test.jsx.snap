// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburger Hamburger laster inn og kan vises 1`] = `undefined`;
